File: pkg/internal/txn/delivery/request.go
package delivery

type POSTTxn struct {
	WUUID     string  `json:"p_w_txnuuid"`
	DUUID     string  `json:"p_d_txnuuid"`
	AUUID     string  `json:"auuid"`
	LUUID     string  `json:"luuid"`
	Reference string  `json:"reference"`
	Email     string  `json:"email" validate:"email"`
	Phone     string  `json:"phone"`
	ToAUUID   string  `json:"toAUUID"`
	Amount    float64 `json:"amount"`
	Concept   string  `json:"concept"`
	Country   string  `json:"country"`
	City      string  `json:"city"`
	Estate    string  `json:"estate"`
	Latitude  float64 `json:"lat"`
	Longitude float64 `json:"lon"`
}

File: cashcoin_mobile_flutter/lib/pkg/internal/txn/repository/api/request.dart
class ReqMake {
  String auuid;
  String reference;
  String toUUID;
  double amount;
  String concept;
  String country;
  String city;
  String estate;
  double latitude;
  double longitude;

  ReqMake({
    required this.auuid,
    required this.toUUID,
    required this.amount,
    required this.concept,
    required this.reference,
    required this.country,
    required this.city,
    required this.estate,
    required this.latitude,
    required this.longitude,});

  Map<String, dynamic> toJson() {
    return {
      'auuid': auuid,
      'toAUUID': toUUID,
      'amount': amount,
      'concept': concept,
      'reference': reference,
      'country': country,
      'city': city,
      'estate': estate,
      'lat': latitude,
      'lon': longitude,
    };
  }
}
